docs/developer_guide.md

```diff
@@ -0,0 +1,145 @@
+
+All of our code are open sourced. Developers can follow this guide to get projects set up.
+
+### Prerequisite
+
+1. Download `qtum-0.14.13-osx64.tar.gz`  from [https://github.com/qtumproject/qtum/releases](https://github.com/qtumproject/qtum/releases) and unzip it.
```

Comment:
Let's not use hard coded versions, ie. `qtum-x.xx.xx-osx64.tar.gz` so this guide doesn't need to be updated all the time.

Reply:
Also, are we gonna have a Windows (and Linux?) guide too?